test/ui/QuerySummaryTest.ts

```diff
@@ -61,6 +62,54 @@ export function QuerySummaryTest() {
       expect($$(test.cmp.element).text()).toEqual(jasmine.stringMatching(/for foo/));
     });
 
+    describe('when there are result lists in the page', () => {
+      let resultListOne: ResultList;
+      let resultListTwo: ResultList;
+
+      beforeEach(() => {
+        resultListOne = new ResultList($$('div', { className: 'CoveoResultList' }).el, null, test.env);
+        resultListTwo = new ResultList($$('div', { className: 'CoveoResultList' }).el, null, test.env);
+        test.cmp.root.appendChild(resultListOne.element);
+        test.cmp.root.appendChild(resultListTwo.element);
+      });
+
+      it('should not display the results range if there is a single result list with infinite scroll enabled', () => {
```

Comment:
I think saying `if there is exactly one result list with...` would be clearer. It could be confuse with the case where you have many result lists and a single one of them has inifinite scroll enabled.